examples/plotting/server/animated.py

```diff
@@ -37,17 +31,14 @@
 # Open the session in a browser
 session.show()
 
-renderer = p.select(dict(type=GlyphRenderer))
-ds = renderer[0].data_source
+ds = p.select_one(dict(name="glyph")).data_source
 
 # These changes to our document will automatically go to the server
 while True:
 
-    rmin = ds.data["inner_radius"]
-    rmin = roll(rmin, 1)
-    rmax = ds.data["outer_radius"]
-    rmax = roll(rmax, -1)
+    rmin = roll(ds.data["inner_radius"], 1)
+    rmax = roll(ds.data["outer_radius"], -1)
 
     ds.data.update(inner_radius=rmin, outer_radius=rmax)
 
-    time.sleep(.10)
+    time.sleep(.03)
```

Comment:
doesn't really relate to your PR but this can be an add_periodic_callback now (and this example doesn't have to be a client anymore, which would speed it up since we'd only be syncing over a single websocket). But add_periodic_callback followed by session.loop_until_closed should also work in a client.